config-models/openconfig/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
I'm not sure if you should be versioning the `Chart.lock`

Reply:
nope